hoot-js/src/main/cpp/hoot/js/JavaScriptTranslator.cpp

```diff
@@ -160,7 +160,7 @@ vector<JavaScriptTranslator::TranslatedFeature> JavaScriptTranslator::_createAll
   QVariantList list)
 {
   vector<TranslatedFeature> result;
-  result.reserve(list.size());
+  //result.reserve(list.size());
```

Comment:
Cool, so this should work on our centos builds w/ qt 4.8, right? Can we uncomment these then?